index.md

```diff
@@ -31,65 +52,68 @@ appropriate choices are made throughout a difficult task.
 
 For our purposes, simulation checklists should help users avoid the
 most common reasons for failure or incorrect results. Checklists will
-typically also be accompanied by an explanation with sources, so the
-rest of the document can go into more detail on best practices and
-cover more possible failure modes and how to avoid them. One can
-divide the types of errors that are made performing molecular
-simulations into (a) mistakes experienced researchers often make, (b)
+typically also be accompanied by a detailed explanation with sources and references (the rationale mentioned above), and this rationale
+can go into more detail on best practices and
+cover more possible failure modes and how to avoid them. 
+
+One can divide the types of errors that are made performing molecular
+simulations into (a) mistakes experienced researchers often make, and (b)
 mistakes new users often make, even after having received solid
 training in fundamentals of molecular simulation. Most checklists will
-be focused on these types of failures.  A third type (c), would
-involve preventing mistakes of naive users who do not understand the
-basic principles of molecular simulation. These will usually require
-special documents aimed at novice users which serve as prerequisites
-for more detailed documents.
-
-**Perpetual Reviews**: 
-Perpetual reviews, like standard reviews, except that they can be
-updated as information comes out, and can be responsive to community
-input, and remain up to date on the current state of the field as it
-evolves. 
+be focused on these types of failures.  Potentially, authors might also be interested in helping to prevent the mistakes of naive users who do not understand the
+basic principles of molecular simulation. However, normal best practices documents can assume a basic familiarity with the fundamentals of molecular simulations; a separate class of documents can be targeted at introducing novice users to the field, and these will serve as prerequisites for more detailed documents. 
+
+### Perpetual Reviews
+ 
+Perpetual reviews, like standard reviews, seek to provide an overview of an area of the field; however, they can be updated as needed, thus serving as living documents that remain current.
+Perpetual reviews change both in response to community input, and in response to new developments in the field.
 An example of this process can be found in the [Mobley et
 al. review](https://github.com/MobleyLab/benchmarksets), which was
-originally an "Annual Reviews in Biophysics" article, but is now being
+originally an *Annual Reviews in Biophysics* article, but is now being
 updated regularly on GitHub.
 
-**Comparisons of Molecular Simulations**:
+### Comparisons of Molecular Simulation Packages and Algorithms:
```

Comment:
I'm not sure about algorimths for now.  That is probably a different thing, especially when getting into areas of efficiency.  Maybe something to bring up?  I'd feel better with just simulation packages for now.